src/controllers/expences.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+const expencesService = require('../services/expences');
+
+const add = (req, res) => {
+  const userData = req.body;
+  const newexpence = expencesService.create(userData);
```

Comment:
Check if userData has all required parameters, otherwise return 400 status. Example: if (!userData.name || !userData.amount) { res.sendStatus(400); return; }